src/functional.rs

```diff
@@ -279,6 +283,82 @@ impl<K, V> Node<K, V> {
             }
         }
     }
+
+    /// Returns a new tree by rotating the right child up to become the root. To maintain the BST
```

Comment:
The BST invariant is just that the left child is less than and the right child is greater than. The AVL invariant asserts that the height of two child nodes of a node never differ in absolute value by more than one. This typo is duplicated for `rorate_right`.
```suggestion
    /// Returns a new tree by rotating the right child up to become the root. To maintain the AVL
```

Reply:
Resolved in 3ca3550